### DISCUSSION TITLE
1.2 Beta 2 Server crashes when looping

### DISCUSSION THREAD

**Discussion by musicgreator** [apohl79/audiogridder#548] [Help]
Windows 10
AMD
Reaper
Using AudioGridder locally on one machine without a VM.

I did a quick test, and had a loop active in Reaper, each time the loop would start again, AudioGridder crashed. I also could not close the plugin GUIs in AG, one GUI was always visible, and I could only switch between different plugins but couldn't close the GUI, except when I closed the AG plugin instance. All these issues aren't present in the 1.1.1 release. 

I hope there will be a focus on keeping AudioGridder compatible with local use! This is useful for many things

**Comment by apohl79** [comment]
I don't follow your description. Please create bug reports for issues especially for crashes. I can't investigate crashes without this.

Regarding the UI, just disable the "keep UI open" setting:

![image](https://user-images.githubusercontent.com/1129160/117821435-e0576200-b26b-11eb-928f-86b4abac97b9.png)

**Comment by musicgreator** [comment]
I simply had a loop active in the daw, that's what crashed it. I will try to make a propper report, but right now it would cost too much time.

**Comment by CaBuffalo** [comment]
Can you make a video of it when you are available?

Thanks.

On Tue, May 11, 2021 at 8:04 AM musicgreator ***@***.***>
wrote:

> I simply had a loop active in the daw, that's what crashed it. I will try
> to make a propper report, but right now it would cost too much time.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/548#discussioncomment-724316>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CVWBSY74BLXX73TSLTNFBRLANCNFSM44U4WRYQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.
